### PR TITLE
Fix e2e login tests for OSE variants

### PR DESCRIPTION
**Pull request by sdodson**
My attempts to run tests locally have all failed so I haven't been able to ensure this works.

### CODE REVIEW

File: assets/test/integration/e2e.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ describe('', function() {
       }, 3000);
 
       expect(browser.driver.getCurrentUrl()).toMatch(/\/login/);
-      expect(browser.driver.getTitle()).toEqual('Login - OpenShift Origin');
```
Comment (liggitt) [inline assets/test/integration/e2e.js]: @jwforres, we'd probably be happy with it starting with `Login - `, right?

Comment (jwforres) [inline assets/test/integration/e2e.js]: Yeah

**Comment by sdodson** [comment]
[test]

File: assets/test/integration/e2e.js (hunk 1 above)
Comment (sdodson) [inline assets/test/integration/e2e.js]: Ok, Updated the match to be more forgiving.

**Comment by sdodson** [comment]
@jwforres look good to you?

**Comment by jwforres** [comment]
yep [merge]

**Pull request merged**